action.yml

```diff
@@ -4,7 +4,7 @@ inputs:
   since-perl:
     description: "List all Perl versions since this (including this). Example: 5.10"
     type: string
-    required: true
+    default: "5.14"
```

Comment:
please leave it as required. This action should not introduce possible failure or missing coverage when default will be changed (and also - what is sane default?)

Reply:
I knew this will be up for debate :-) 

The goal was trying to make the workflow easier to use, and I would say whatever is the minimal Perl version toolchain decide support could/should be the default. Of course when you are using the default, the pros and cons is that version is going to change on your behalf. This is still leaving the ability to anyone to explicitly set a minimum version.

The problem I was trying to solve is to let everyone put an arbitrary version number.
Let me know if I've not convinced you, and will revert that change

Reply:
personally I'm against default version with tag `v1`

there can be branch / tag `v1-514` which will contain wrapper over this action providing default version (or different workflow). Still it should be something which will signal, that different default than 5.14 is mistake.

Reply:
ok going to preserve the required argument for now